index.html

```diff
@@ -539,7 +539,120 @@ <h2>
         issuance protocol is identified by a [=digital credential/protocol
         identifier=]. See also section [[[#protocol-registry]]].
       </dd>
+      <dt>
+        Digital credential coordinator
+      </dt>
+      <dd>
+        See [=credential request coordinator=].
+      </dd>
     </dl><!--
+    // MARK: Credential Request Coordinator
+    -->
+    <h2>
+      Credential Request Coordinator
+    </h2>
+    <p>
+      The <dfn data-local-lt="coordinator">credential request coordinator</dfn>
+      is a user-agent-defined component that mediates [=digital credential=]
+      interactions within a [=top-level traversable=]. Each [=top-level
+      traversable=] has exactly one associated coordinator. The coordinator
+      ensures that at most one interaction is active across all [=child
+      navigables=], orchestrates the end-to-end flow of presentation or
+      issuance, and manages transitions between [=credential request
+      coordinator/interaction states=]. A user agent may delegate some or all
+      coordinator responsibilities to external wallet applications, platform
+      components, or other trusted entities according to user or platform
+      policy.
+    </p>
+    <p>
+      The coordinator manages the lifecycle of the interaction's {{Promise}}
+      and its associated {{AbortSignal}}, including resolution with the user's
+      selected [=digital credential/digital credential|digital credential(s)=]
+      or rejection due to errors or cancellation by user or program.
+    </p>
+    <p>
+      The [=credential request coordinator=]:
+    </p>
+    <ul>
+      <li>Validates and transforms presentation or issuance inputs and outputs.
+      </li>
+      <li>Interprets each individual request in the input list as targeting a
+      specific [=holder=], and prepares [=holder=] options accordingly.
+      </li>
+      <li>Presents the available [=holders=] to the user for selection.
+      </li>
+      <li>Resolves the interaction's {{Promise}} with the selected [=digital
+      credential=] or rejects it to indicate cancellation.
+      </li>
+    </ul>
+    <aside class="note">
+      <p>
+        Although the coordinator handles input/output coordination, the
+        user-selected [=holder=] typically acts as the [=credential chooser=].
```

Comment:
```suggestion
        Although the coordinator handles input/output coordination,
        the user-selected [=holder=] acts as the [=credential chooser=]. Alternatively,
        the platform presents the [=credential chooser=] with credentials gathered from various [=holders=].
```

Reply:
I would like to keep text at a more abstract level. User-selected holder hints that there is a holder selection screen is presented. I am proposing a more abstract text that leaves the details of coordination out of the spec.
What do you think?

```suggestion
Although the coordinator handles input/output coordination, it is the responsibility of the platform together with available [=holders=], to present the [=credential chooser=].
```

Reply:
Tweaked to highlight @mohamedamir's changes. More tweaking will be required on the `together with available [=holders=]` phrase to make it proper English, if the meaning of this existing phrasing is accepted.
```suggestion
        Although the coordinator handles input/output coordination, it is the
        responsibility of the platform together with available [=holders=], to present the [=credential chooser=].
```